pages*: improve list placeholders and wording of descriptions (#12111)
tldr-pages/tldr: pages/windows/findstr.md

> Find specified text within one or more files.
> More information: <https://learn.microsoft.com/windows-server/administration/windows-commands/findstr>.

> - Find one or more in all files:

Reviewer: ```suggestion
- Find one or more strings in all files:
```

`findstr "{{string1 string2 ...}}" *`

> - Find one or more string(s) in a piped command's output:

Reviewer: ```suggestion
- Find one or more strings in a piped command's output:
```

`{{dir}} | findstr "{{string1 string2 ...}}"`

> - Find one or more string(s) in all files recur[s]ively:

Reviewer: ```suggestion
- Find one or more strings in all files recur[s]ively:
```
